packages/prosess-tilkjent-ytelse/src/components/feriepenger/FeriepengerPanel.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FunctionComponent } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Undertittel } from 'nav-frontend-typografi';
+import {
+  ArbeidsgiverOpplysningerPerId,
+  Feriepengegrunnlag,
+  FeriepengegrunnlagAndel,
+  KodeverkMedNavn,
+} from '@fpsak-frontend/types';
+import { VerticalSpacer } from '@fpsak-frontend/shared-components';
+import { Column, Row } from 'nav-frontend-grid';
+import { EkspanderbartpanelBase } from 'nav-frontend-ekspanderbartpanel';
+import FeriepengerPrAar from './FeriepengerPrAar';
+
+const finnListeMedOpptjeningsår = (andeler: FeriepengegrunnlagAndel[]): number[] => {
+  const årsliste = andeler.map((andel) => andel.opptjeningsår).sort((a, b) => a - b);
+  return [...new Set(årsliste)];
+};
+
+const hentTittel = (): React.ReactNode => (
+  <Undertittel>
+    <FormattedMessage id="TilkjentYtelse.Feriepenger.Tittel" />
+  </Undertittel>
+);
+
+interface OwnProps {
+  feriepengegrunnlag: Feriepengegrunnlag;
+  alleKodeverk: {[key: string]: KodeverkMedNavn[]};
+  arbeidsgiverOpplysningerPerId: ArbeidsgiverOpplysningerPerId;
+  erPanelÅpent: boolean;
+  togglePanel: any;
+}
+
+export const FeriepengerPanel: FunctionComponent<OwnProps> = ({
+  feriepengegrunnlag, alleKodeverk, arbeidsgiverOpplysningerPerId, erPanelÅpent, togglePanel,
+}) => {
+  const { andeler } = feriepengegrunnlag;
+  if (!andeler || andeler.length < 1) {
+    return null;
+  }
+  const opptjeningsår = finnListeMedOpptjeningsår(andeler);
```

Comment:
Her kan ein med fordel bruka useMemo for å unngå å køyra funksjon ved kvar render